### PR TITLE
cleanup items for LU generation

### LINKED ISSUE #3374: LU Generation tweaks
A few cleanup items for LU generation.
1) Stop generating the <form>-trigger lu file.  It is not used.
2) Make all entity labels be `{<entity>=}` rather than `{@<entity>=}`.
3) Change ADD LABELS HERE to 'ADD AND LABEL INCORRECT UTTERANCES HERE`

### PR DESCRIPTION
**Pull request by coldplaying42**
Fixes #3374 

## Proposed Changes

  - 
  -
  -

### CODE REVIEW

File: experimental/generation/generator/packages/library/templates/generator.lg [hunk 1]
```diff
@@ -155,7 +155,7 @@
 
 # actionAssign(entity)
 > Assign entity to current property
-- ${list(where([actionSetOrPush(`@${entity}`), actionDeletePropertyToChange()], actions, actions != ''))}
+- ${list(where([actionSetOrPush(`${entity}`), actionDeletePropertyToChange()], actions, actions != ''))}
```
Comment (chrimc62) [inline experimental/generation/generator/packages/library/templates/generator.lg:159]: This tells me you have not tried to actually tested a generated dialog.  I always build sandwich and interact with it to make sure scenarios work.  Removing @ is only for labels in LU files.  Here this is how you refer to an entity and it should be here.

File: experimental/generation/generator/packages/library/templates/professional/form-CrossTrain.en-us.lg [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 # qnaAnswer_text()
 > Show QnA answer
-- ${@answer}
+- ${answer}
```
Comment (chrimc62) [inline experimental/generation/generator/packages/library/templates/professional/form-CrossTrain.en-us.lg:11]: This file should not be changed.

**Comment by chrimc62** [comment]
You should always test to see if things work.  This does not work because @ is how you refer to entities at runtime. This change is only about removing the @ in the LU files--not in .dialog/trigger files.

**Review by chrimc62 — CHANGES_REQUESTED** [review]
:clock1:

File: experimental/generation/generator/packages/library/test/merge.test.ts [hunk 3]
```diff
@@ -344,7 +344,7 @@ describe('dialog:generate --merge files', async function () {
             await assertMissing('language-generation/en-us/BreadValue/sandwichMerge-BreadValue.en-us.lg', /white/, errors)
             //sandwichMerge
             await assertMissing('language-understanding/en-us/sandwichMerge.en-us.lu', /pulled/, errors)
-            await assertContains('language-understanding/en-us/Bread/sandwichMerge-Bread-BreadValue.en-us.lu', />- {@BreadProperty={@BreadValue=rye}}/, errors)
+            await assertContains('language-understanding/en-us/Bread/sandwichMerge-Bread-BreadValue.en-us.lu', />- {@BreadProperty={BreadValue=rye}}/, errors)
```
Comment (chrimc62) [inline experimental/generation/generator/packages/library/test/merge.test.ts:347]: >{@ [](http://example.com/codeflow?start=115&length=2)

There should not be an @ here--this test should be failing.

File: experimental/generation/generator/packages/library/test/merge_data/modified/language-understanding/en-us/Bread/sandwichMerge-Bread-BreadValue.en-us.lu [hunk 4]
```diff
@@ -60,120 +60,120 @@
...
-- {@BreadProperty={@BreadValue=grain} bread}
-- {@BreadProperty={@BreadValue=wheat} bread}
-- {@BreadProperty={@BreadValue=multi grain} bread}
-- {@BreadProperty={@BreadValue=grain wheat} bread}
-- {@BreadProperty={@BreadValue=multi grain wheat} bread}
->- {@BreadProperty={@BreadValue=rye} bread}
-- {@BreadProperty={@BreadValue=white} bread}
-- {@BreadProperty={@BreadValue=whole} bread}
-- {@BreadProperty={@BreadValue=whole wheat} bread}
+- {@BreadProperty={BreadValue=multi}}
```
Comment (chrimc62) [inline experimental/generation/generator/packages/library/test/merge_data/modified/language-understanding/en-us/Bread/sandwichMerge-Bread-BreadValue.en-us.lu:63]: >{@ [](http://example.com/codeflow?start=2&length=2)

This should not be there.  This is the file where all labels should not include an @.

File: experimental/generation/generator/packages/library/templates/generator.lg (hunk 1 above)
Comment (coldplaying42) [reply experimental/generation/generator/packages/library/templates/generator.lg:159]: You mean the whole generator.lg should not be changed?

**Review by chrimc62 — APPROVED** [review]
:shipit:

**Pull request merged**